src/config/base.rs

```diff
@@ -139,3 +140,39 @@ impl Base {
         }
     }
 }
+
+static DEFAULT_TEST_REGION: &str = "127.0.0.1:19000";
+
+fn parse_region(maybe_region: Option<String>) -> s3::Region {
```

Comment:
This is currently only called from `Base::test()` so the unwrap below is fine. I'm sure we're not entirely consistent, but when using unwrap/expect we try to include a doc comment with a `# Panics` section so that it is clear this shouldn't be called from production code.

(I think we have a few exceptions where we can panic in startup code if the configuration is invalid, so that exception would likely apply here as well. Just that in particular, we try to never risk a panic in middleware/endpoints, even though those should just result in a status 500.)